core/src/main/scala/org/apache/spark/storage/ShuffleBlockFetcherIterator.scala

```diff
@@ -1261,6 +1262,10 @@ final class ShuffleBlockFetcherIterator(
     }
   }
 
+  private[this] def addFailureFetchResult(result: FailureFetchResult): Unit = {
```

Comment:
Is it still necessary to extract a method?

Reply:
removed the method